src/Microsoft.DocAsCode.Build.JavaScriptReference/AutoMapper/ApiLanguageValuePairTypeConverter.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.JavaScriptReference
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using AutoMapper;
+
+    public class ApiLanguageValuePairTypeConverter : ITypeConverter<string, List<ApiLanguageValuePair>>
+    {
+        private readonly string[] _supportedLanguages;
+
+        public ApiLanguageValuePairTypeConverter(string[] supportedLanguages)
+        {
+            if (supportedLanguages == null)
+            {
+                throw new ArgumentNullException(nameof(supportedLanguages));
+            }
+            if (supportedLanguages.Length == 0)
+            {
+                throw new ArgumentException($"{nameof(supportedLanguages)} cannot be empty");
+            }
+            _supportedLanguages = supportedLanguages;
+        }
+
+        public List<ApiLanguageValuePair> Convert(string source, List<ApiLanguageValuePair> destination, ResolutionContext context)
```

Comment:
Oh I found it [https://github.com/AutoMapper/AutoMapper/wiki/Custom-type-converters](https://github.com/AutoMapper/AutoMapper/wiki/Custom-type-converters)...

---
In reply to: [93909005](https://github.com/dotnet/docfx/pull/1061#discussion_r93909005) [](ancestors = 93909005)